src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/config.jelly

```diff
@@ -187,6 +187,17 @@
                         help="/plugin/azure-vm-agents/help-enableMSI.html">
                         <f:checkbox/>
                     </f:entry>
+
+                    <f:section title="${%Title_UAMI}">
+                        <f:entry title="${%Enable_UAMI}" field="enableUAMI" help="/plugin/azure-vm-agents/help-enableUAMI.html">  
+                            <f:checkbox/>
+                        </f:entry>
+
+                        <f:entry title="${%UAMI_ID}" field="uamiID">
```

Comment:
Should we provide a help Text with the structure of the expected id here? I first thought this may be the objectId. Eventually, I had a look at the code to figure out what needs to be inserted here.